src/Cache/Interceptors/AgentInterceptor.php

```diff
@@ -7,17 +7,23 @@
 
 class AgentInterceptor extends Interceptor
 {
-
-    private const AGENT = "php:0.5.0";
     private bool $isFirstRequest = true;
+    private string $agent;
+    private string $runtimeVersion;
+
+    public function __construct(string $clientType)
+    {
+        $this->agent = sprintf("php:%s:0.5.0", $clientType);
```

Comment:
Can you please change this to 1.10.0 for now so we're closer to the truth? I'm going to set up release please for this repo soon and let it stamp the version into the code, but I'd like to go ahead and update this while we're in here.